sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -456,6 +456,63 @@ public async Task StartCopyFromUriAsync_Error()
             }
         }
 
+        [Ignore("Dependent on set tier")]
```

Comment:
Is this going to get fixed in the next day or so?  If not, it'd be good to file a bug to track that we need to come back and update this test.